### PR TITLE
moving demos to docs

### PR DESCRIPTION
**Pull request by visagim**

### CODE REVIEW

File: docs/demos/rabi.ipynb [hunk 1]
```diff
@@ -571,6 +571,7 @@
     "    fig.suptitle(title_label)\n",
     "    return fig, axes\n",
     "\n",
+    "\n",
```
Comment (AlbertMitjans) [inline docs/demos/rabi.ipynb:574]: Was this on purpose?

**Review by AlbertMitjans — APPROVED** [review]
Thanks!!

File: docs/demos/rabi.ipynb (hunk 1 above)
Comment (visagim) [reply docs/demos/rabi.ipynb:574]: nope

Comment (visagim) [reply docs/demos/rabi.ipynb]: apparently it was nbqa-black doing some funny sfuf

**Review by fedonman — APPROVED** [review]

**Pull request merged**